pkg/cache/informer_cache_test.go

```diff
@@ -0,0 +1,28 @@
+package cache_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/client-go/rest"
+
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+var _ = Describe("informerCache", func() {
+	It("should not require LeaderElection", func() {
+		cfg := &rest.Config{}
+
+		mapper, err := apiutil.NewDynamicRESTMapper(cfg, apiutil.WithLazyDiscovery)
+		Expect(err).ToNot(HaveOccurred())
+
+		c, err := cache.New(cfg, cache.Options{Mapper: mapper})
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
I couldn't find a matcher for "type X implements interface Y", the closes to that I found is `BeAssignableToTypeOf` but that errors out if one of the two is nil.

Reply:
this is clear enough.  If you're really worried, you can always add an explanation after the "be true" part as a format string, but I think this is readable enough for now.